### PR TITLE
Update FAQ to reflect CSI GA

### PR DESCRIPTION
**Pull request by saad-ali**

### CODE REVIEW

File: sig-storage/volume-plugin-faq.md [hunk 1]
```diff
@@ -12,11 +12,12 @@ A *Kubernetes Volume plugin* extends the Kubernetes volume interface to support
 **How do I implement a Volume plugin?**
 
 There are three methods to implement a volume plugin:
-1. In-tree volume plugin
-2. Out-of-tree FlexVolume driver
+1. In-tree volume plugin [deprecated]
```
Comment (msau42) [inline sig-storage/volume-plugin-faq.md]: do we need to make an announcement in the release notes that it's deprecated?

Comment (saad-ali) [reply sig-storage/volume-plugin-faq.md]: The existing drivers will continue to work and are not deprecated. Adding new volumes is deprecated, and this is documenting that.

**Pull request merged**